### PR TITLE
redis use YAML to store

### PR DESCRIPTION
**Pull request by sinneduy**

### CODE REVIEW

**Comment by nikhilmat** [comment]
Any idea how to test this? Also, do you need to `require 'yaml'`? not sure if it's required elsewhere, but it needs to be required in standard library

File: lib/gemstash/cache.rb [hunk 1]
```diff
@@ -93,15 +92,19 @@ def alive!
       @cache.ping == "PONG"
     end
 
+    def get(key)
+      @cache.get(YAML.load(key))
```
Comment (nikhilmat) [inline lib/gemstash/cache.rb]: @sinneduy this should be:
`YAML.load(@cache.get(key))`

Comment (nikhilmat) [reply lib/gemstash/cache.rb]: you also probably need to guard against nil

Comment (nikhilmat) [reply lib/gemstash/cache.rb]:
```
irb(main):002:0> YAML.load(nil)
TypeError: no implicit conversion of nil into String
```

File: lib/gemstash/cache.rb [hunk 2]
```diff
@@ -93,15 +92,19 @@ def alive!
       @cache.ping == "PONG"
     end
 
+    def get(key)
+      YAML.load(@cache.get(key)) unless @cache.get(key).nil?
```
Comment (nikhilmat) [inline lib/gemstash/cache.rb]: might as well make this a single network call:
```
value = @cache.get(key)
YAML.load(value) if value
```

**Comment by sinneduy** [comment]
looks like it didn't need the `require 'yaml'`.  However, not too sure how we can write tests, I'm not sure we have a great example in the existing gemstash

**Pull request merged**